src/eventMapping.ts

```diff
@@ -104,43 +104,45 @@ export function getCloudEvent(req: express.Request): CloudEventsContext | null {
     return cloudevent;
   }
 
-  console.log('Converting from background event to CloudEvent');
   const event = getBackgroundEvent(req);
-  if (event === null) {
-    console.error('Unable to extract background event');
-    return null;
+  if (!event) {
+    throw new Error('Unable to extract background event');
   }
 
   const context = event.context;
   const data = event.data;
 
+  cloudevent = {
+    contenttype: CE_CONTENT_TYPE,
+    id: context.eventId,
+    specversion: CE_SPEC_VERSION,
+    time: context.timestamp,
+    data: data,
+  };
```

Comment:
The logic of `cloudevent` in this function, `getCloudEvent` is confusing because we have `cloudevent` as a let and we have multiple returns..

I think we should try to remove `cloudevent`. Keeping multiple returns is fine, but could be shorter. Example:

```ts
export function getCloudEvent(req: express.Request): CloudEventsContext {
  // Handle a CloudEvent in binary content mode.
  if (isBinaryCloudEvent(req)) {
    return {
      ...getBinaryCloudEventContext(req),
      data: req.body
    };
  }

  // Handle a CloudEvent in structured content mode.
  if (isStructuredCloudEvent(req)) {
    return req.body as CloudEventsContext;
  }

  const event = getBackgroundEvent(req);
  if (!event) {
    throw new Error('Unable to extract background event');
  }

```